quandl/__init__.py

```diff
@@ -6,5 +6,38 @@
 
 from .model.database import Database
 from .model.dataset import Dataset
+from .model.datatable import Datatable
 from .model.data import Data
 from .model.merged_dataset import MergedDataset
+import warnings
+import copy, os
+
+def get_table(code, **options):
+    paginate = None
+    if 'paginate' in options.keys(): paginate = options.pop('paginate')
+    row_limit = 100
+    data = None
+    while True:
+        next_options = copy.deepcopy(options)
+        next_data = Datatable(code).data(params=next_options)
+
+        if data is None:
+            data = next_data
+        else:
+            data.extend(next_data)
+
+        if len(data.values) >= row_limit:
+            if os.isatty(0):
+                warnings.warn("data over row limit, rest data won't include", UserWarning)
+            break
+
+        next_cursor_id = next_data.meta['next_cursor_id']
+        if next_cursor_id is None:
+            break
+        elif paginate is not True and next_cursor_id is not None:
+            if os.isatty(0):
+                warnings.warn("This is the first page, for more pages, please use 'paginate=True'", UserWarning)
```

Comment:
Can you keep a list of each of the messages that need to be reviewed and put the list in a new ticket in JIRA to be addressed later.